tests/openthread_ncp/README.md

```diff
@@ -0,0 +1,43 @@
+## OpenThread on RIOT
+
+This test demonstrates how to use the [OpenThread](https://github.com/openthread/openthread)
+open source implementation of [Thread](https://threadgroup.org/) on RIOT.
+
+This test shows how to use a Network Co-Processor (NCP). A Network Co-Processor is used with wpantund software.
+Wpantund is a user-space network interface driver/daemon that provides a native IPv6 network interface to a low-power
+wireless Network Co-Processor. NCP and wpantund communicates by UART (UART(0) is used here)
+
+## Quick usage
+
+To test OpenThread NCP in RIOT, you can do the following:
+
+1. Flash one node with openthread_ncp example with `make BOARD=<target> clean all flash -C tests/openthread_ncp`
+2. Install wpantund: On Ubuntu 16.04, run
+```
+sudo apt-get -y install build-essential subversion libncurses5-dev libssl-dev zlib1g-dev gawk gcc-multilib flex git-core gettext gcc binutils bzip2 python perl make unzip libz-dev tftp git shtool autogen automake libtool autotools-dev libdbus-1-dev
+git clone https://github.com/openthread/wpantund.git
+cd wpantund
+./bootstrap.sh
+./configure
+make
+sudo make install
```

Comment:
I'm not a big fan of this kind of command line. Better use the `--prefix` of the configure option and pass the configuration file from the `wpantund` command line (use `-c` option)